sys/dev/aic7xxx/aic79xx.c

```diff
@@ -6261,8 +6264,32 @@ ahd_init(struct ahd_softc *ahd)
 	/*
 	 * Only allow target mode features if this unit has them enabled.
 	 */
-	if ((AHD_TMODE_ENABLE & (0x1 << ahd->unit)) == 0)
+#ifdef AHD_TARGET_MODE
+        if (resource_int_value(device_get_name(ahd->dev_softc),
+                               device_get_unit(ahd->dev_softc),
+                               "tmode_enable", &tmode_enable) != 0) {
+		if (bootverbose)
+			printf("%s: No ahd.%i.tmode_enable hint found\n",
+				ahd_name(ahd), ahd->unit);
```

Comment:
I think it is too verbose to report every hint that was not set in a system.

Reply:
The thinking there was to alert users who used to use `AHC_TMODE_ENABLE` that something had changed. But I see your point. If I keep the `AHC_TMODE_ENABLE` in addition to the new hint I guess that won't be a problem.